RobotFramework_TestsuitesManagement/Config/CConfig.py

```diff
@@ -522,7 +522,8 @@ def __loadConfigFileLevel2(self):
             raise Exception
         if self.sConfigName not in oSuiteConfig:
             CConfig.sLoadedCfgError = f"Testsuite management - Loading configuration level 2 failed! \n \
```

Comment:
Hi Son,
please confirm the proper look, or if a concatenation would be a better solution for the line-break.
Thank you,
 Thomas

Reply:
Hi Thomas,

I have just updated log output of variants list. Now the log look like below:
[ ERROR ] Testsuite management - Loading configuration level 2 failed! 
         The variant 'variant_23' is not defined in 'C:\Users\mas2hc\Desktop\data_storage\workspace\ROBFWAIO-OSS\robotframework-testsuitesmanagement\atest\general_config\testsuites_config.json'
         Please find the suitable variant in this list:
            - 'default'
            - 'variant_1'
            - 'variant_2'

Thank you,
Son